beesdoo_product/models/beesdoo_product.py

```diff
@@ -210,26 +211,16 @@ def _compute_total(self):
                 "beesdoo_product.consignes_group_tax", raise_if_not_found=False
             )
 
-            taxes_included = set(product.taxes_id.mapped("price_include"))
+            taxes_included = set(product.taxes_id.filtered(lambda t: t.tax_group_id != consignes_group).mapped("price_include"))
             if len(taxes_included) == 0:
                 product.total_with_vat = product.list_price
                 return True
 
             elif len(taxes_included) > 1:
-                raise ValidationError(
-                    _("Several tax strategies (price_include) defined for %s")
-                    % product.name
-                )
```

Comment:
I think that writing that into the log file may be a good idea instead of simply dropping the error. Isn't it ?

Reply:
I'm not sure it's needed since the price with vat will be 0.0 you know that something went wrong and since it's for the end user maybe we could have a second computed field error_message that we set and we display it in the view directly